erlang/src/erl_parser.mly

```diff
@@ -103,6 +105,7 @@ HASH
 %token <Parse_info.t> EOF
 
 %start <Erl_ast.structure> module_file
+%start <Erl_ast.expr list> exprs_file
```

Comment:
This entrypoint be called `terms` instead to better indicate what we intend to parse with it